plugins/koterm.koplugin/profile

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+# This file gets executed on every start of the KOTerm shell.
+# Do not edit this file!
+# You may change the contents of 'aliases' and 'profile.user'
+#
+
+PATH="${PATH}:$(pwd)/plugins/koterm.koplugin/scripts"
+TERM="vt52"
+
+PS1="$ "
+if [ "$(id -u)" -eq "0" ]; then
+    PS1="# "
+fi
+
+export PATH
+export TERM
+export PS1
+
+# Source a user profile if it exists.
+USER_PROFILE="$(pwd)/plugins/koterm.koplugin/scripts/profile.user"
+if [ -f "${USER_PROFILE}" ]; then
+    "${USER_PROFILE}"
```

Comment:
Oh, usually the suggested alternative for source is, my bad for not mentioning that.
```suggestion
    . "${USER_PROFILE}"
```